profile.md

```diff
@@ -16,8 +16,10 @@ or REQUIRED level requirements defined herein.
 - [Originating Identity Header](#originating-identity-header)
 - [Context Object](#context-object)
   - [Context Object Properties](#context-object-properties)
-  - [Cloud Foundry](#cloud-foundry)
-  - [Kubernetes](#kubernetes)
+  - [Cloud Foundry Context Object](#cloud-foundry-context-object)
+  - [Kubernetes Context Object](#kubernetes-context-object)
+- [Bind Resource Object](#bind-resource-object)
+  - [Cloud Foundry Bind Resource Object](#cloud-foundry-bind-resource-object)
```

Comment:
I really like that you added the extra words ("Context Object", "Bind Resource Object") to these titles. Just makes it easier to track what section you're in 
👍

Reply:
Having said that, something is a bit off with this TOC. In some cases we list level 4 stuff but not level 3 stuff.  And I think the platform specific context object stuff might be one level too deep.  If you don't want to fix those in this PR that's fine, I'll do it in a follow-on one- just thought I mention it before I forget.

Reply:
Yeah I noticed that. I'll just fix it in here whilst it's open as has no LGTXs.

Reply:
OK, PR updated. Now all level 2 and 3 titles are in the TOC.